### PR TITLE
Miscellaneous quality of life improvements to Grants

### PR DESCRIPTION
**Pull request by jalehman**

### CODE REVIEW

File: styles/markdown.module.css [hunk 1]
```diff
@@ -95,12 +95,6 @@
   @apply py-2 ml-6 text-base md:text-lg lg:text-xl;
 }
 
-.markdown li a,
-.markdown h3 a,
-.markdown h2 a {
-  @apply text-green;
-}
-
```
Comment (matildepark) [inline styles/markdown.module.css]: ???

Comment (jalehman) [reply styles/markdown.module.css]: where did that that come from? definitely didn't edit this directly—is it generated and not supposed to be committed?

**Comment by g-a-v-i-n** [comment]
Screen Shot 2021-07-28 at 1 36 30 PM

This works but is still visually confusing. @urcades was going to do a refactor of this whole UI I think. Correct me if I'm wrong, Ed

File: styles/markdown.module.css (hunk 1 above)
Comment (matildepark) [reply styles/markdown.module.css]: You should retain that css to keep all the links green inside posts — currently inside headers or lists links just have no styling

**Comment by urcades** [comment]
@g-a-v-i-n yeah, absolutely — will get some suggestions floated into here so we can implement asap

File: styles/markdown.module.css (hunk 1 above)
Comment (jalehman) [reply styles/markdown.module.css]: Whoops. Added back.

**Comment by jalehman** [comment]
> @g-a-v-i-n yeah, absolutely — will get some suggestions floated into here so we can implement asap

I'm going to just release this, and we can update the design afterwards.

**Pull request merged**